LongWarOfTheChosen/Localization/XComGame.rus

```diff
@@ -4844,7 +5339,37 @@ LocLongDescription="Предоставляет дополнительное дв
 LocFriendlyName="Улучшенная командная работа"
 LocHelpText="Предоставляет дополнительное действие. Число зарядов может быть один или два, в зависимости от уровня связи напарников."
 LocLongDescription="Предоставляет дополнительное действие. Число зарядов может быть один или два, в зависимости от уровня связи напарников."
-; End Translation
+; End Translated
+
+; LWOTC Translated
+[Shadow X2AbilityTemplate]
+LocLongDescription="Активирует Тень на 1 ход"
+LocHelpText="Жнец укрывается в тени, особой маскировке, на 1 ход."
+LocPromotionPopupText="<Bullet/> Тень уменьшает радиус обнаружения на 70%.<br/><Bullet/> Reaper has increased mobility while in Shadow.<br/><Bullet/> Уйти в Тень свободное действие и перезаряжется 4 хода.<br/>"
+
+[Sting X2AbilityTemplate]
+LocPromotionPopupText="<Bullet/> Жало позволяет атаковать Жнецу, оставаясь в Тени.<br/><Bullet/> используется только 1 раз за ход во время пребывания в Тени и расходует <Ability:SelfAmmoCost/> боезапаса.<br/>"
+
+[SilentKiller X2AbilityTemplate]
+LocLongDescription="Гарантирует Жнецу остаться в Тени при убийстве."
+LocHelpText="Гарантирует Жнецу остаться в Тени при убийстве с основного оружия."
+LocPromotionPopupText="<Bullet/> Позволяет Жнец оставаться в Тени, если убить противника с основного оружия."
+
+[ShadowRising X2AbilityTemplate]
+LocLongDescription="Уменьшает перезарядку тени на 1 ход."
+LocHelpText="Уменьшает перезарядку тени на 1 ход."
+LocPromotionPopupText="<Bullet/> .<br/>"
+; End Translated
+
+
+; LWOTC Translated
+[BloodTrail X2AbilityTemplate]
+LocFriendlyName="Кровавый след"
+LocLongDescription="Выстрел делает +<Ability:BLOODTRAILDAMAGE/> урона если цель была уже ранена на этом ходу. Добавляет кровотечение, не суммируется."
+LocHelpText=="Выстрел делает +<Ability:BLOODTRAILDAMAGE/> урона если цель была уже ранена на этом ходу. Добавляет кровотечение, не суммируется."
+
+LocPromotionPopupText="<Bullet/> Действует только на тот ход, когда был ранен противник.<br/><Bullet/> Планируйте наперед, чтобы использовать преимущество кровавого следа.<br/><Bullet/> Кровотечение наносит 1 урон в течении 2 ходов.<br/><Bullet/> Последующие выстрелы лишь обновляют время действия кровотечения до 2 ходов.<br/><Bullet/> Кровотечение так же работает и при броске ножей.<br/>"
+; End Translated
```

Comment:
Some of these [will change shortly](https://github.com/long-war-2/lwotc/pull/1178/files#diff-0c3559cce935fc36aef53dcc259c82aeacc54c87769f5883b5fe3cd0d51b32d2L5235).